<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberManage</class>
 <widget class="QWidget" name="MemberManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MemberManage</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 24pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>  会员管理</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>人员列表：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>侯子童</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>何雨泊</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>杨晨</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>汪恒辉</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>会员列表：</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_vip">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>设为会员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_vip">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>删除会员</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
